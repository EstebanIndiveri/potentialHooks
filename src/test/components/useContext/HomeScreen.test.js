import React from 'react';
import '@testing-library/jest-dom';
import { mount } from 'enzyme';
import { UserContext } from '../../../components/useContext.js/UserContext';
const { default: HomeScreen } = require("../../../components/useContext.js/HomeScreen");

    describe('pruebas con <HomeScreen/>', () => {
        const user={
            name:'fernan',
            email:'gmai.com'
        }
        const wrapper=mount(
            <UserContext.Provider value={{user}}>
                <HomeScreen/>
            </UserContext.Provider>
        )
        test('should ', () => {
            expect(wrapper).toMatchSnapshot();
        })
        
    });
    