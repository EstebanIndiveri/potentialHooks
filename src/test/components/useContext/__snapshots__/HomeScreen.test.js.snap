// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas con <HomeScreen/> should  1`] = `
<div>
  <h1>
    HomeSreen
  </h1>
  <hr />
  <pre>
    {
   "name": "fernan",
   "email": "gmai.com"
}
  </pre>
</div>
`;
